README.md

```diff
@@ -7,9 +7,61 @@
 An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
 for developing state-of-the-art deep learning models on a wide variety of linguistic tasks.
 
-## Quickstart
+## Running AllenNLP
+
+The preferred way to install AllenNLP is via `pip`.
```

Comment:
I'd also consider just saying something like "Just run `pip install allennlp` in your python 3.6 environment, install pytorch 0.3.1 from their website (link), and you're good to go!  If you need pointers on setting up a python 3.6 environment, see below."